rlkit/core/rl_algorithm.py

```diff
@@ -429,12 +431,14 @@ def evaluate(self, epoch):
             self.env.reset_task(idx)
             paths = []
             # TODO: put these lines into method evaluate_task(idx) that goes into sac.py?
+            # (AZ): should this be hardcoded?
```

Comment:
it's just for debugging, so I think it's fine. but we should probably either put a comment that it's for debugging, or only run this code in debug mode...

Reply:
wait is this one actually for debuggin? it looks like this is how we're evaluating train tasks?

Reply:
it's the way we evaluate train tasks to judge if we're overfitting to the encoder replay buffer during training.